macho_parser/docs/LC_SEGMENT_64.md

```diff
@@ -88,3 +88,11 @@ The difference between `+load` and `__mod_init_func` is that the former guarante
 
 ## __LINKEDIT
 `__LINKEDIT` segment contains data that's used by the linker. Unlike other segments, this one doesn't have sections. Its contents are described by other load commands, e.g. `LC_SYMTAB`, `LC_DYSYMTAB`.
+
+## __LLVM_COV
+`__LINKEDIT` is used for code coverage.
+
+### __llvm_covfun
```

Comment:
Was this intentional? Asking to learn.

Reply:
Yes. There are two sections under `__LLVM_COV`, `__llvm_covfun` and `__llvm_covmap`. I added both for completeness, and probably will learn this later.